iic-odk-auth/Controllers/AuthController.cs

```diff
@@ -19,10 +19,12 @@ public class AuthController : Controller
     private readonly ILogger<AuthController> _logger;
     private readonly IOdkService odkService;
     private readonly String salt;
+    private readonly Configuration _config;
```

Comment:
can you please add whitelistEmails as private variable instead of _config and set its value in constructor